pandas/tests/io/test_orc.py

```diff
@@ -224,3 +225,50 @@ def test_orc_reader_snappy_compressed(dirpath):
     got = read_orc(inputfile).iloc[:10]
 
     tm.assert_equal(expected, got)
+
+
+def test_orc_roundtrip_file(dirpath):
+    # GH44554
+    # PyArrow gained ORC write support with the current argument order
+    pytest.importorskip("pyarrow", minversion="7.0.0")
```

Comment:
put the decorator on the function itself

Reply:
Hmm looks like it may not do what it is meant to:
https://github.com/pytest-dev/pytest/issues/9542
https://github.com/pytest-dev/pytest/issues/9548

We don't really want to skip all other ORC tests for pyarrow version below 7.0.0 since the ORC reader is about 2 years older than the writer.

Reply:
like this
```
pandas/tests/io/test_parquet.py:    @td.skip_if_no("pyarrow", min_version="1.0.0")
```

we have this as a standard pattern

Reply:
@jreback  Really thanks! Fixed!